### ISSUE TITLE
Add modifiable hyperparameters.

### ISSUE THREAD

**Issue by ShayanRiyaz** [MLH-Fellowship/neuro-art#14]
Our model uses multiple hyperparameters for the Neural Style Transfer Model to work:
- We want to add these hyperparameters as available tuning options for our website.
Some of the ones used are:
- [ ] Style weight
- [ ] Content weight
- [ ] Learning rate Alpha for the Adam Optimizer.
- [ ] Epochs
- [ ] Steps per epoch
